config/packages/prod/Centreon.yaml

```diff
@@ -195,22 +195,57 @@ services:
 
     # Monitoring resources
     _instanceof:
-        Centreon\Infrastructure\Monitoring\Resource\Provider\ProviderInterface:
-            tags: ['monitoring.resource.providers']
+        Centreon\Infrastructure\Monitoring\MonitoringResource\Repository\Provider\ProviderInterface:
+            tags: ['monitoring.monitoringresource.repository.providers']
             calls:
                 - method: setSqlRequestTranslator
                   arguments: ['@sqlRequestTranslator']
+        Centreon\Domain\Monitoring\MonitoringResource\Interfaces\HyperMediaProviderInterface:
+            tags: ['monitoring.monitoringresource.hypermediaproviders']
+        Centreon\Infrastructure\Monitoring\MonitoringResource\API\v2110\Validator\Interfaces\MonitoringResourceValidatorInterface:
+            tags: ['monitoring.monitoringresource.api.validators']
 
-    Centreon\Domain\Monitoring\Interfaces\ResourceServiceInterface:
-        class: Centreon\Domain\Monitoring\ResourceService
-
-    Centreon\Domain\Monitoring\Interfaces\ResourceRepositoryInterface:
-        class: Centreon\Infrastructure\Monitoring\Resource\ResourceRepositoryRDB
+    Centreon\Domain\Monitoring\MonitoringResource\Interfaces\MonitoringResourceRepositoryInterface:
+        class: Centreon\Infrastructure\Monitoring\MonitoringResource\Repository\MonitoringResourceRepositoryRDB
         calls:
             - method: setSqlRequestTranslator
               arguments: ['@sqlRequestTranslator']
             - method: setProviders
-              arguments: [!tagged_iterator 'monitoring.resource.providers']
+              arguments: [!tagged_iterator 'monitoring.monitoringresource.repository.providers']
+
+    Centreon\Application\Controller\MonitoringResourceController:
+        calls:
+            - method: setHyperMediaProviders
+              arguments: [!tagged_iterator 'monitoring.monitoringresource.hypermediaproviders']
+
+    Centreon\Domain\Monitoring\MonitoringResource\UseCase\v2110\FindMonitoringResources\FindMonitoringResources:
+        class: Centreon\Domain\Monitoring\MonitoringResource\UseCase\v2110\FindMonitoringResources\FindMonitoringResources
+
+    Centreon\Domain\Monitoring\MonitoringResource\UseCase\v2110\DetailHostMonitoringResource\DetailHostMonitoringResource:
+        class: Centreon\Domain\Monitoring\MonitoringResource\UseCase\v2110\DetailHostMonitoringResource\DetailHostMonitoringResource
+
+    Centreon\Domain\Monitoring\MonitoringResource\UseCase\v2110\DetailServiceMonitoringResource\DetailServiceMonitoringResource:
+        class: Centreon\Domain\Monitoring\MonitoringResource\UseCase\v2110\DetailServiceMonitoringResource\DetailServiceMonitoringResource
+
+    Centreon\Domain\Monitoring\MonitoringResource\UseCase\v2110\DetailMetaServiceMonitoringResource\DetailMetaServiceMonitoringResource:
+        class: Centreon\Domain\Monitoring\MonitoringResource\UseCase\v2110\DetailMetaServiceMonitoringResource\DetailMetaServiceMonitoringResource
```

Comment:
IMO Not needed as you don't autowire interface or define particular methods / arguments in your configuration.